colors_script/calc_colormap.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import sys
+sys.path.append('../mindboggle/mio')
```

Comment:
i would suggest not doing this but doing through a standard import call.

also, you can look into `click` to update the mindboggle command line to support more options.